stateful/es-data-svc.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
why is this needed?

Reply:
From the link I provided above:
"StatefulSets currently require a Headless Service to be responsible for the network identity of the Pods. You are responsible for creating this Service."

When trying to deploy a statefulset without specifying an existing service name, it returns with an error (the one in the #220 issue)

That's why this fix adds a service for each statefulset and label selectors because:
"In 1.8 and later versions, failing to specify a matching Pod Selector will result in a validation error during StatefulSet creation."